RestApi/Sepes.RestApi/Startup.cs

```diff
@@ -30,10 +42,44 @@ public void ConfigureServices(IServiceCollection services)
             
             // The following line enables Application Insights telemetry collection.
             //TODO add support for local logging. If this is left empty then no logs are made. Unknown if still affects performance.
-            //Add support for other means of registring the key
-            services.AddApplicationInsightsTelemetry(Configuration["AzureLogToken:ServiceApiKey"]); 
-            
+            //Secret key can be set up for with either secret key or in appsettings.json, secret key will overwrite json.
+            services.AddApplicationInsightsTelemetry(Configuration["AzureLogToken:ServiceApiKey"]);
+            services.Configure<AppSettings>(Configuration.GetSection("Jwt"));
+            services.AddOptions();
+
+            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)  
+            .AddJwtBearer(options =>  
+            {  
+            options.TokenValidationParameters = new TokenValidationParameters  
+                {  
+                ValidateIssuer = false,  //TODO set to true before MVP
```

Comment:
Small tings like this that are easy to forget should realy be GitHub issues.

Reply:
Sure, I will get it done before my last commit